chart/templates/_helpers.tpl

```diff
@@ -8,6 +8,11 @@ app: shell
 release: {{ .Release.Name }}
 {{- end }}
 
+{{- define "elastic.release_labels" }}
+app: elastic
```

Comment:
I would rather use the existing `drupal.release_labels` and extend those in the deployment definition, the same way that we are doing it for the Drupal deployment.

Reply:
Same as https://github.com/wunderio/drupal-project-k8s/commit/041607b1c4bfdd62a4aba1f4f949d6c4fdf7130c